dkc/core/rest/user.py

```diff
@@ -9,3 +13,17 @@ class Meta:
             'id',
             'username',
         ]
+
+
+class UserViewSet(GenericViewSet):
+    queryset = User.objects.all()
+
+    permission_classes = [AllowAny]
```

Comment:
In the future, we may want the user list to not be available to all, but this is fine for now.

Reply:
Nevermind, I didn't realize it was a `GenericViewSet`, and doesn't provide any default endpoints.